java/display-scene-in-tabletop-ar/README.md

```diff
@@ -17,7 +17,7 @@ NOTE: [Google Play Services for AR](https://play.google.com/store/apps/details?i
 ## How it works
 
 1. Create an `ArcGISARView`.
-2. Listen for a tap from the user, then use `mArView.setInitialTransformationMatrix(screenPoint)` to set the initial transformation, which allows you to place the scene. This method uses ARKit's built-in plane detection.
+2. Listen for a tap from the user, then use `mArView.setInitialTransformationMatrix(screenPoint)` to set the initial transformation, which allows you to place the scene. This method uses ARCore's built-in plane detection.
 3. Create and display the scene. To allow you to look at the content from below, set the base surface navigation constraint to `NONE`.
 4. To create a realistic tabletop scene experience, set the scene's base surface opacity to 0. This will ensure that only the target scene content is visible.
 5. For tabletop scenes, the `arView`'s `originCamera` must be set such that the altitude of the camera matches the altitude of the lowest point in the scene. Otherwise, scene content will float above or below the targeted anchor position identified by the user. For this sample, the origin camera's latitude and longitude are set to the center of the scene for best results. This will give the impression that the scene is centered on the location the user tapped.
```

Comment:
ticky tacky, there is no "arView" variable, we have mArView, which is referenced in the new step 2.

Reply:
good catch!